solr_conf/conf/solrconfig.xml

```diff
@@ -11,10 +11,10 @@
   <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.StandardDirectoryFactory}"/>
 
   <!-- solr lib dirs -->
```

Comment:
To add to this, by default Jetty does not use your solr conf stuff! It will simply download the archive from `Downloading jetty at https://github.com/projecthydra/hydra-jetty/archive/v8.2.1.zip` which has its own solr/fedora configs (for solr its here https://github.com/samvera/hydra-jetty/tree/v8.2.1/solr).

If you want to use your own you can use the jetty:config (which i have never done before until today).